source/conf.py

```diff
@@ -59,6 +61,21 @@
 #
 html_theme = 'alabaster'
 
+version_list_var = os.getenv('VERSION_LIST')
+extra_nav_links = {'Live Version': '/index.html'} # default link to Live version
+
+if version_list_var is not None:
+    version_list = version_list_var.split(',')
+    for ver in version_list:
+        extra_nav_links[f'Version {ver}'] = f'/versions/{ver}/index.html'
+
+print(extra_nav_links)
```

Comment:
Is this just for debugging?

Reply:
Yes, I think it's useful to print this for inclusion in the CI logs.